token-swap/program/src/processor.rs

```diff
@@ -2086,7 +2086,7 @@ mod tests {
             ) = accounts.setup_token_accounts(
                 &user_key,
                 &withdrawer_key,
-                initial_a,
+                withdraw_amount,
```

Comment:
This change seems unrelated, in here by accident?

Reply:
No, CI fails without this change on `Err(Custom(1))` instead of the expected `Err(Custom(3))`. This is because I changed  the order of error-checks. Updating the initial_a amount ensures that we are making it to the mint check.